src/frontend/packages/lib_classroom/src/components/DashboardClassroomStudent/index.spec.tsx

```diff
@@ -52,6 +52,8 @@ describe('<DashboardClassroomStudent />', () => {
       />,
     );
 
+    screen.getByText(classroom.title!);
```

Comment:
What you can do to avoid using the non-null assertion operator, is to set a `title` and a `description` in the factory call and use the text here.